framework/ui/build.gradle

```diff
@@ -93,6 +93,7 @@ dependencies {
     implementation libs.androidx.navigation.compose
     implementation libs.compose.ui.ui
     implementation libs.compose.material3
+    implementation libs.accompanist.flowlayout
```

Comment:
I would prefer to avoid experimental or alpha dependencies to reduce maintenance. Keep it simple, if the tags don't fit in one line either ellipsis them or use a lazyRow or even a LazyGrid.